projects/plugins/boost/app/modules/Modules_Index.php

```diff
@@ -24,7 +24,7 @@ class Modules_Index {
 	protected $modules = array();
 
 	/**
-	 * @var Pluggable[] - Classes that handle all Jetpack Boost featues.
+	 * @var class-string<Pluggable>[] - Classes that handle all Jetpack Boost featues.
```

Comment:
How does this work?

```suggestion
	 * @var class-string<Pluggable>[] - Classes that handle all Jetpack Boost features.
```

Reply:
`class-string` specifies a string that is the name of a class (or, I suppose, it could be an interface).

`class-string<Pluggable>` further specifies that the named classes extend/implement `Pluggable`.

Then `[]` indicates that it's an array of such strings.

Reply:
I guess it helps if I look up PHPStan docs instead of just PHPDoc and Phan:
https://phpstan.org/writing-php-code/phpdoc-types#class-string

Reply:
Phan's are a little disorganized, true. It's mentioned on https://github.com/phan/phan/wiki/About-Union-Types.

Something to be careful when looking at phpstan's docs is that Phan doesn't support everything phpstan does.